flathub.json

```diff
@@ -0,0 +1,3 @@
+{ 
+    "only-arches": ["x86_64"] 
```

Comment:
Any reason to skip aarch64?

Reply:
Originally PySIde2 was only compiled for x86_64, but you're right, there shouldn't be any reason it not being compilable on aarch64.

Reply:
Fixed in latest commit.